JASP-Engine/JASP/R/equivalencebayesianpairedsamplesttest.R

```diff
@@ -243,3 +250,72 @@ EquivalenceBayesianPairedSamplesTTest <- function(jaspResults, dataset, options)
   
   return()
 }
+
+.densityPriorPosteriorPairedTTestTable <- function(jaspResults, dataset, options, equivalenceBayesianPairedTTestResults, ready) {
+  
+  # Create table
+  equivalenceDensityPairedTTestTable <- createJaspTable(title = gettext("Equivalence Density Table"))
+  equivalenceDensityPairedTTestTable$dependOn(c("pairs", "lowerbound", "upperbound", "priorWidth", "effectSizeStandardized","informative", "informativeCauchyLocation", "informativeCauchyScale",
+                                                 "informativeNormalMean", "informativeNormalStd", "informativeTLocation",
+                                                 "informativeTScale", "informativeTDf"))
+  equivalenceDensityPairedTTestTable$showSpecifiedColumnsOnly <- TRUE
+  
+  # Add Columns to table
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "variable1",   title = " ",                         type = "string")
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "separator",   title = " ",                         type = "separator")
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "variable2",   title = " ",                         type = "string")
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "section",     title = "Section",                   type = "string")
+  equivalenceDensityPairedTTestTable$addColumnInfo(name = "density",     title = "Density",                   type = "number")
+  
+  if (ready)
+    equivalenceDensityPairedTTestTable$setExpectedSize(length(options$pairs))
+  
+  jaspResults[["equivalenceDensityPairedTTestTable"]] <- equivalenceDensityPairedTTestTable
+  
+  if (!ready)
+    return()
+  
+  .equivalenceDensityFillPairedTableMain(equivalenceDensityPairedTTestTable, dataset, options, equivalenceBayesianPairedTTestResults)
+  
+  return()
+}
+
+.equivalenceDensityFillPairedTableMain <- function(equivalenceDensityPairedTTestTable, dataset, options, equivalenceBayesianPairedTTestResults) {
+  
+  for (pair in options$pairs) {
+    
+    namePair <- paste(pair[[1L]], " - ",  pair[[2L]], sep = "")
+    results <- equivalenceBayesianPairedTTestResults[[namePair]]
+    
+    if (!is.null(results$status)) {
+      equivalenceDensityPairedTTestTable$addFootnote(message = results$errorFootnotes, rowNames = namePair, colNames = "density")
+      equivalenceDensityPairedTTestTable$addRows(list(variable1 = pair[[1L]], separator = "-", variable2 = pair[[2L]], density = NaN), rowNames = namePair)
+    } else {
+      equivalenceDensityPairedTTestTable$addRows(list(variable1     = pair[[1L]], 
+                                                      separator     = "-",
+                                                      variable2     = pair[[2L]],
+                                                      section       = "p(\U003B4 \U02208 I | H1)",
```

Comment:
So maybe we should think of how to translate this... perhaps something like:

```r
gettextf("p(%1$s I | H1, y)", "\U003B4 \U02208")
```
?
On the other hand that might be a bit unnecessary for this release (it will be the same in Dutch).